modules/wmi/charts.go

```diff
@@ -821,49 +807,60 @@ func (w *WMI) updateCharts(mx *metrics) {
 	w.updateLogonCharts(mx)
 	w.updateThermalzoneCharts(mx)
 	w.updateTCPCharts(mx)
-	w.updateAppsCharts(mx)
+	w.updateProcessesCharts(mx)
 }
 
-func (w *WMI) updateAppsCharts(mx *metrics) {
-	if !mx.hasApps() {
+func (w *WMI) updateProcessesCharts(mx *metrics) {
+	if !mx.hasProcesses() {
 		return
 	}
+
 	if !w.cache.collectors[collectorProcess] {
 		w.cache.collectors[collectorProcess] = true
-		if err := w.Charts().Add(newAppsCharts()...); err != nil {
+
+		if err := w.Charts().Add(newProcessesCharts()...); err != nil {
 			w.Warning(err)
 		}
 	}
-	for _, apps := range mx.Apps.info {
-		if w.cache.apps[apps.ID] {
+
+	procs := make([]string, 0, len(mx.Processes.procs))
+	for _, proc := range mx.Processes.procs {
+		procs = append(procs, proc.ID)
+	}
+	sort.Slice(procs, func(i, j int) bool { return procs[i] < procs[j] })
+
+	for _, id := range procs {
+		if w.cache.procs[id] {
```

Comment:
Why did you abbreviate `process` ?

Reply:
Because I decided to do so - procs/processes. Does it look like a problem for you?

Reply:
No, it is not a problem.

When I brought abbreviation for `apps`, it was because this is the way we work on Linux/BSD/Mac. On the other hand we are not abbreviating `memory` when we could.

I am still trying to understand the pattern. :thinking:

Reply:
> w.cache.**procs**

That is a private variable. Why does it bother you how I name it?

Reply:
Or do you talk about something different? I am commenting on the highlighted code.

Reply:
I understand this is an internal variable, but why should it to be the unique with abbreviation when others do not have? :thinking: